src/strategies/genomesdao/index.ts

```diff
@@ -21,27 +22,67 @@ export async function strategy(
   snapshot
 ): Promise<Record<string, number>> {
   const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const votingAddress = "0x48D1CcB09f771788F59c8aAAB613936eDfA267b7"; // voting contract
 
   const multi = new Multicaller(network, provider, abi, { blockTag });
-  addresses.forEach((address: any) =>
-    multi.call(address, options.address, 'balanceOf', [address])
+
+  const result = await erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
   );
```

Comment:
Hello again @Charles-repo what i mean was use `erc20-balance-of` strategy in your strategies of your space, the result will be sum of all these strategies

Reply:
@ChaituVR , thanks for your kind answer.
For our case, we need to get the balance for 5 different contracts by address.
How to get these with one time call of `erc20-balance-of`?

Reply:
@Charles-repo you can add 5 `erc20-balance-of` strategies in your space settings

Reply:
@ChaituVR Could you please provide me an example to use space?

Reply:
![image](https://user-images.githubusercontent.com/15967809/161414476-b520b7d6-c53e-4eb8-9155-5159aa677dde.png)
I mean like this @Charles-repo In your space

Reply:
@ChaituVR I may misunderstand you, but in our case, we use different weight for each balance of tokens.

Reply:
Hi @Charles-repo You mean like (balance of token 1 * 10) + (balance of token 2 * 20 + .. etc? 

you can try using https://snapshot.org/#/strategy/erc20-balance-of-weighted instead of erc20-balance-of